modules/events/events-page/layout/sidebar/timeline-filter/SortButton.tsx

```diff
@@ -1,12 +1,21 @@
+import { multipleCategoriesRemoved } from '@/modules/events/events.multifilter.slice'
+import { eventsPerPageChanged } from '@/modules/events/events.slice'
+import { useAppDispatch } from '@/modules/redux/app/hooks'
 import React from 'react'
-import Button from '@mui/material/Button';
 
 type Props = {}
 
 export default function SortButton({ }: Props) {
+    const dispatch = useAppDispatch()
+
+    const handleClick = () => {
+        dispatch(multipleCategoriesRemoved())
+        dispatch(eventsPerPageChanged(10))
```

Comment:
This is not consistent with sorting. I suggest changing the title of the Button to something like 'Reset'

Reply:
Yeah btw, since our sorting happens when a user clicks on one of the action then we should remove the sort button.